src/lib.rs

```diff
@@ -1653,6 +1653,96 @@ impl Bindings {
             ))
         }
     }
+    
+    /// Write these bindings as source text to stdout.
+    pub fn write_to_stdout(&self) -> io::Result<()> {
+        let bindings = self.to_string();
+        
+        match self.rustfmt_generated_string(bindings) {
+            Ok(Some((rustfmt_bindings, rustfmt_stderr))) => { 
+                println!("{}", rustfmt_bindings);
+                eprintln!("{}", rustfmt_stderr);
+            },
+            Ok(None) => (),
+            Err(err) => println!("{:?}", err),
+        }
+        
+        Ok(())
+    }
+    
+    /// Checks if rustfmt_bindings is set and runs rustfmt on the string
+    fn rustfmt_generated_string(&self, source: String) -> io::Result<Option<(String, String)>> {
+        let _t = self.context.timer("rustfmt_generated_string");
+
+        if !self.context.options().rustfmt_bindings {
+            return Ok(None);
+        }
+
+        let rustfmt = if let Ok(rustfmt) = which::which("rustfmt") {
+            rustfmt
+        } else {
+            return Err(io::Error::new(
+                io::ErrorKind::Other,
+                "Rustfmt activated, but it could not be found in global path.",
+            ));
```

Comment:
This should just return `Ok(source)` as well. We don't want to avoid emitting bindings just because there is no `rustfmt` (`rustfmt`ing is configured on by default).

We can `eprintln!("warning: could not find usable rustfmt to pretty print bindings")` to give users an idea why the bindings aren't pretty printed.